src/components/Cart/CartItems/CartItems.jsx

```diff
@@ -31,6 +41,47 @@ export const CartItems = () => {
     }
   };
 
+  const inWishlist = (_id) => {
+    const productInWishlist = wishlistState.wishlist.find(
+      (product) => product._id === _id
+    );
+
+    return productInWishlist ? true : false;
+  };
+
+  const listProducts = async () => {
+    try {
+      const response = await axios.get("/api/products");
+      if (response.status === 200) {
+        setProducts(response.data.products);
+      } else {
+        throw new Error();
+      }
+    } catch (e) {
```

Comment:
You could name the `e` as `error`. Because I'm not able to make out whether it is an event or an error. A proper naming would help others understand code better.

Reply:
Done